setup_windows.py

```diff
@@ -12,8 +12,8 @@ def get_config():
 
     extra_objects = []
 
-    # client = "mysqlclient"
-    client = "mariadbclient"
+    client = "mysqlclient"
+    # client = "mariadbclient"
```

Comment:
Don't do that. MySQL Connector/C 6.1 is too old, and new libmysqlclient doesn't support static linking.

Reply:
The whole reason I landed on 6.1 was because that is what the build was looking for. I aimed for making the minimum changes possible for it to work.  Prior to these changes the build fails because it is searching for a 'mariadb' directory  inside the 'MySQL Connector C 6.1' directory (that is caused by line 16), and the 'MySQL Connector C 6.1' directory if installed today lands in the x64 'Program Files' directory

Reply:
MySQL Connector/C 6.1 is old product.  It is not recommended/default option. So don't try make it success without modification.